docs/user-guide/src/bmo/raid.md

```diff
@@ -0,0 +1,169 @@
+# RAID setup
+
+[RAID](https://en.wikipedia.org/wiki/RAID) is a technology that allows creating
+volumes with certain properties out of 2 or more physical disks. Depending on
+the [RAID level](https://en.wikipedia.org/wiki/Standard_RAID_levels), you may
+be able to merge several disks into a larger one or achieve redundancy.
+
+Metal3 supports two RAID implementation:
+
+- *Hardware RAID* is implemented by hardware itself and can be configured
+  through the machine's BMC.
+- *Software RAID* is implemented by the Linux kernel and can be configured
+  using the standard `mdadm` tool.
+
+To create or delete RAID volumes, you need to edit the `spec.raid` field of the
+`BareMetalHost` resource, changing either the `hardwareRAIDVolumes` or the
+`softwareRAIDVolumes` array. If the host is in the `available` state, it will
+be moved to the `preparing` state and the new settings will be applied. After
+some time, the host will move back to `available`, and the resulting changes
+will be reflected in its `status.raid` field.
+
+**Note:** RAID setup requires 1-2 reboots of the machine and thus may take 5-20
+minutes.
+
+**Warning:** never try to configure both hardware and software RAID at the same
+time on the same host. While theoretically possible, this mode makes little
+sense and is not supported well by the underlying Ironic service.
+
+## Hardware RAID
+
+*Hardware RAID* is a type of RAID that is configured by a special component of
+the bare-metal machine - *RAID controller*. The resulting RAID volumes are
+normally presented transparently to the operating system and can be used as
+normal disks.
+
+Not all hardware models and Metal3 drivers support RAID: check [supported
+hardware](supported_hardware.md) for details.
+
+### Automatic allocation
+
+One approach is to define the required level, disk count and volume size,
+letting Ironic to automatically select the disks to place RAID on, for example:
+
+```yaml
+spec:
+  raid:
+    hardwareRAIDVolumes:
+    - name: volume1
+      level: "5"
+      numberOfPhysicalDisks: 3
+      sizeGibibytes: 350
+```
+
+The most common RAID levels are `0`, `1`, `5` and `1+0`. Levels `2`, `6`,
+`5+0` and `6+0` are also supported by Metal3 but may not be supported by all
+hardware models. The level dictates the minimum number of physical disks and
+the maximum size of a RAID volume.
+
+**Note:** because of values like `1+0`, RAID level is a string, not a number.
+
+You can use the boolean `rotational` field to filter out physical disks that
```

Comment:
Can we elaborate a bit on this. Does setting rotational to true mean the raid is only created from rotational disks, and false it is created from ssds, and absense of the value (null) that any will be used?